cl_dll/hud_spectator.cpp

```diff
@@ -1988,3 +1989,33 @@ void CHudSpectator::InitHUDData()
 	gHUD.m_iFOV =  CVAR_GET_FLOAT("default_fov");
 }
 
+CHudSpectator::OverviewLayer CHudSpectator::GetCurrentLayer(Vector playerPos)
+{
+	// Get the layer with highest Z, that will be the default layer,
+	// or it will be the dummy oneif an overview file doesn't exist for this map
+	OverviewLayer result = GetHighestLayer();
+
+	// Get the overview with highest Z that is below player Z position too
+	for (const auto& layer : m_OverviewData.layers)
+	{
+		if (playerPos.z < layer.z && layer.z < result.z)
```

Comment:
This gets the overview with the lowest Z that is above player Z position, directly contrary to the comment

Reply:
Thanks, I'll update the comment. It was for a different approach that I had and forgot to change the comment accordingly.